compiler/src/iree/compiler/Codegen/Common/CPU/CPUMaterializeEncodingPass.cpp

```diff
@@ -134,9 +134,9 @@ enumerateMatmulTileArm64(TypeRange elementTypes, ExecutableTargetAttr target) {
     }
   }
 
-  if (hasUkernel(target) && lhs.isSignlessInteger(8) &&
-      rhs.isSignlessInteger(4) && out.isSignlessInteger(32)) {
-    if (hasFeature(target, "+i8mm")) {
+  if (lhs.isSignlessInteger(8) && rhs.isSignlessInteger(4) &&
+      out.isSignlessInteger(32)) {
+      if (hasFeature(target, "+i8mm")) {
       return {
           TileMxNxK{4, 8, 16},
```

Comment:
Good! But now we need to combine this with the changes we had in the earlier state of this PR. This latest state has reverted them. Meaning, this `{4, 8, 16}` needs to be replaced by what there is in the ukernels `.inl` file.

Reply:
oh it is already the same with the ukernel tile sizes: 
https://github.com/iree-org/iree/blob/daee1dd35d52c69f08a02266757f1dac63434b9e/runtime/src/iree/builtins/ukernel/arch/arm_64/mmt4d_arm_64_tiles.inl#L59

Reply:
Oh - I see.